src/OrchardCore.Modules/OrchardCore.Contents/Views/Content.PublishButton.cshtml

```diff
@@ -1,22 +1,27 @@
-@*@if (Authorizer.Authorize(Permissions.PublishContent, (IContent)Model.ContentItem)) {*@
+@using Microsoft.AspNetCore.Authorization
+@inject IAuthorizationService AuthorizationService
+
 @{
+    ContentItem contentItem = Model.ContentItem;
     var returnUrl = Context.Request.Query["returnUrl"];
+    var hasPublishOwnContentPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.PublishOwnContent, contentItem);
```

Comment:
Please read the code comments @hishamco 

```
    public class CommonPermissions
    {
        // Note - in code you should demand PublishContent, EditContent, or DeleteContent
        // Do not demand the "Own" variations - those are applied automatically when you demand the main ones

        // EditOwn is the permission that is ultimately required to create new content. See how the Create() method is implemented in the AdminController
```

Reply:
Aha, I think I read a similar comment, but m question is how the non `xxxOwn` permission verify if the the current user is owner of the content, just to understand and satisfy myself

Thanks

Reply:
Can you update this pr to remove all checks for Own variants, so we can review what it is doing. The PermissionHandler will morph permission checks as required to validate that the user is allowed access to the requested resource.

Reply:
I will remove the own checks if it's handled behind the scene